lib/server/pipeline/filter/sample.rb

```diff
@@ -23,9 +23,13 @@ def call(payload)
 
         private
 
-        def ignore?(payload)
-          return false if @platform.empty?
-          !@platform.include?(payload[:platform])
+        def match_platform?(payload)
+          return true if @platform.empty?
+          @platform.include?(payload[:platform])
+        end
+
+        def no_errors?(payload)
+          !payload[:errors] || payload[:errors].empty?
```

Comment:
Can we make this configurable as the platform?

Reply:
Done as requested.